pkg/monitor/serialization/serialize.go

```diff
@@ -144,9 +144,10 @@ func IntervalsToFile(filename string, intervals monitorapi.Intervals) error {
 func EventsIntervalsToJSON(events monitorapi.Intervals) ([]byte, error) {
 	outputEvents := []EventInterval{}
 	for _, curr := range events {
-		if curr.From == curr.To && !curr.To.IsZero() {
+		if !curr.Display && curr.From == curr.To && !curr.To.IsZero() {
```

Comment:
Double checking.  Seems like we would want !curr.Display ||( curr.From == curr.To && !curr.To.IsZero() )

Currently this will allow 0 length intervals that don't have curr.Display set to false, is that the intention?

Reply:
Nice, yes I think you're right, thanks!

Reply:
pushed